README.md

```diff
@@ -126,13 +126,66 @@ The following is an example `membersUpdate` event received by listeners when a u
 ]
 ```
 
+### Members
+
+Members is used to build avatar stacks and find out which members are online within a space.
+
+```ts
+// Subscribe to all member events in a space by passing the 'update' event
+space.members.subscribe('update', (memberUpdate) => {
+  console.log(memberUpdate);
+});
+
+// Subscribe to when a member enters the space
+space.members.subscribe('enter', (memberJoined) => {
+  console.log(memberJoined);
+});
+
+// Subscribe to when a member leaves the space
+space.members.subscribe('leave', (memberLeft) => {
+  console.log(memberLeft);
+});
```

Comment:
There's also a [remove event](https://github.com/ably-labs/spaces/blob/main/src/Members.ts#L17) which is emitted when a member is considered [removed from the space](https://github.com/ably-labs/spaces/blob/main/src/Members.ts#L138) (i.e. when they left the space more than [120s](https://github.com/ably-labs/spaces/blob/main/src/Space.ts#L23) ago and have yet to return).